templates/common/infra/bicep/core/database/postgresql/flexibleserver.bicep

```diff
@@ -0,0 +1,42 @@
+param name string
+param location string = resourceGroup().location
+param tags object = {}
+
+param sku object
+param storage object
+param administratorLogin string
+@secure()
+param administratorLoginPassword string
+param databaseNames array = []
+param enableFirewall bool = false
+
+// PostgreSQL version
+param version string
+
+resource postgresServer 'Microsoft.DBforPostgreSQL/flexibleServers@2022-01-20-preview' = {
+  location: location
+  tags: tags
+  name: name
+  sku: sku
+  properties: {
+    version: version
+    administratorLogin: administratorLogin
+    administratorLoginPassword: administratorLoginPassword
+    storage: storage
+    highAvailability: {
+      mode: 'Disabled'
+    }
+  }
+
+  resource database 'databases' = [for name in databaseNames: {
```

Comment:
I changed this to allow any number of database names, since the dapr sample didn't specify any (just used the built in named postgres) and I could imagine other samples creating multiple.